internal/app/public.go

```diff
@@ -39,6 +39,33 @@ func publicAPI(
 		return c.JSON(http.StatusOK, output)
 	})
 
+	r.POST("/signup/verify", func(c echo.Context) error {
+		var inp http1.CreateVerificationInput
+		if err := c.Bind(&inp); err != nil {
+			return &echo.HTTPError{Code: http.StatusBadRequest, Message: fmt.Errorf("failed to parse request body: %w", err)}
+		}
+		output, err := controller.CreateVerification(c.Request().Context(), inp)
+		if err != nil {
+			return err
+		}
+
+		return c.JSON(http.StatusOK, output)
+	})
+
+	r.GET("/signup/verify/:code", func(c echo.Context) error {
```

Comment:
```suggestion
	r.POST("/signup/verify/:code", func(c echo.Context) error {
```

- HTTP `GET` should not cause any side effects
- Often, when you click the link on the email, you will see a screen saying that the verification is complete, but let's leave this to the frontend. The backend simply returns JSON, and the frontend calls this API when the screen is displayed. (On the other hand, if you want to display this screen in the backend, you need to use GET so that the browser can display it. This is a special case of GET that causes side effects)